install.bat

```diff
@@ -507,8 +548,19 @@ local function get_msvc_env_setup_cmd()
 	assert(type(vars.UNAME_M) == "string", "requires vars.UNAME_M to be set before calling this function.")
 	local x64 = vars.UNAME_M=="x86_64"
 
-	-- 1. try visual studio command line tools
-	local vcdir = get_visual_studio_directory()
+	-- 1. try visual studio command line tools of VS 2017 or higher
+	local vsdir = get_visual_studio_directory_from_vswhere()

```

Comment:
Should report any error messages encountered by this function.

Reply:
Added in commit https://github.com/luarocks/luarocks/pull/1040/commits/98f7f4d9b5287d5a9e76715f3039e1983fca1b23